ci-operator/step-registry/dpu-operator/workflow/tests/dpu-operator-workflow-tests-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: dpu-operator-workflow-tests
+  from_image:
+    namespace: ci
+    name: telco-runner
```

Comment:
telco-runner?

Reply:
Do we have some level of transparency what this container image looks like?

Reply:
yes! Here it is: https://github.com/openshift/release/blob/master/clusters/app.ci/supplemental-ci-images/telco-runner.yaml

Reply:
Please put that in the same README.md